src/Psr/CacheItemPool/CacheItem.php

```diff
@@ -106,7 +106,7 @@ public function setIsHit(bool $isHit): self
     /**
      * {@inheritdoc}
      */
-    public function set($value): CacheItemInterface
+    public function set($value): static
```

Comment:
Can we also add argument types so we are aligned with the interface?

Reply:
added (needed to drop "psr/cache": "^1.0  for it to work)

Reply:
@boesing if you want the parameter types to match the interface, we will need a new major version here.

If we can keep `psr/cache:^1` and BC, then we can release a minor.